src/styles/main.css

```diff
@@ -0,0 +1,103 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 406px;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+}
+
+.card__desc {
+  margin-top: 40px;
```

Comment:
Better to use margin-bottom instead m-top to avoid margin collapse. Let's fix in all cases